.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
No, only VSCode will pick up these changes. They are really just some settings to override the rules for line lengths (since we have pretty long lines) 

There were a lot of warnings otherwise. PEP-8 declares you shouldn't have longer than 80 but many people are okay with 120.